src/main/java/lotto/view/InputView.java

```diff
@@ -1,21 +1,58 @@
 package lotto.view;
 
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 import java.util.Scanner;
 import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 public class InputView {
 
     private final static Scanner scanner = new Scanner(System.in);
 
     public static int requestPrice() {
         System.out.println("구입급액을 입력해주세요.");
-        int num = Integer.parseInt(scanner.nextLine().trim());
-        return num;
+        try {
+            return Integer.parseInt(scanner.nextLine().trim());
+        }catch (NumberFormatException e){
+            System.out.println("입력을 확인해주세요.");
+            return requestPrice();
+        }
     }
 
-    public static List<Integer> requestNums() {
+    public static List<List<Integer>> requestManualNumsList() {
+        System.out.println("\n수동으로 구매할 로또 수를 입력해 주세요.");
+        int count = 0;
+        try {
+            count = Integer.parseInt(scanner.nextLine().trim());
+            if(count < 0){
+               throw new NumberFormatException();
+            }
+        }catch (NumberFormatException e){
```

Comment:
코드 작성 후 IntelliJ 등의 코드 자동 정렬 기능을 이용해 정렬을 맞춰주시면 좋을 것 같습니다!

만약 번거로우시면 IntelliJ의 Save action 등의 플러그인을 통해 저장 시 자동으로 줄을 맞춰주는 기능을 이용하실수도 있고

Git의 pre commit hooks 등을 걸어서 커밋 전에 코드 정렬을 한번 해주게 구현하실수도 있어요

```suggestion
        } catch (NumberFormatException e){
```